frontend/public/components/dashboard/inventory-card/inventory-item.tsx

```diff
@@ -62,35 +64,28 @@ export const InventoryItem: React.FC<InventoryItemProps> = React.memo(
   }
 );
 
-export const Status: React.FC<StatusProps> = React.memo(({ groupID, count }) => {
```

Comment:
Note: this component is effectively unused so it was removed.

Reply:
I think we'll need it for #2399

Reply:
But we can bring it back in a follow-on if necessary

Reply:
If this is not being used anywhere else I can remove its usage from #2399. @rawagner what do you think?

Reply:
@bipuladh but its used in https://github.com/openshift/console/pull/2399 so we dont want to remove it. The component was used at some point by metal3 inventory card but they removed the usage recently https://github.com/openshift/console/commit/28a956da3737413e07c78bffef803de20c1334ae . It should be back for 4.3. I'd keep `Status` as is.

Reply:
Thanks everyone for clarification, I'll keep the `Status` component as we plan to use it in other PRs.